models.py

```diff
@@ -78,6 +78,16 @@ class Marker(MarkerMixin, Base): # TODO rename to AccidentMarker
     address = Column(Text)
     locationAccuracy = Column(Integer)
 
+    # Addition required to split description (Omer):
```

Comment:
The fields should be `Integer` columns, since otherwise they blow up the database and they're not flexible if we want to change the way they are displayed later.